stubs/vobject/vobject/behavior.pyi

```diff
@@ -1,10 +1,11 @@
-from typing import Any
+from _typeshed import Incomplete
+from typing import Any, ClassVar
 
 class Behavior:
     name: str
     description: str
     versionString: str
-    knownChildren: Any
+    knownChildren: ClassVar[dict[str, tuple[int, int | None, Incomplete | None]]]
```

Comment:
I'm pretty sure from the docstring that the third element is also meant to be `int | None`: https://github.com/py-vobject/vobject/blob/f7d9a5a7c8c999387dbc2028bdf5041842327953/vobject/behavior.py#L26-L30

```suggestion
    knownChildren: ClassVar[dict[str, tuple[int, int | None, int | None]]]
```